refinery/core/models.py

```diff
@@ -2213,3 +2220,87 @@ class SiteProfile(models.Model):
 
     def __unicode__(self):
         return self.site.name
+
+
+class SiteStatistics(models.Model):
+    """
+    A model to encapsulate various information about a deployed Refinery
+    Instance's utilization.
+    """
+    datasets_shared = models.IntegerField()
+    datasets_uploaded = models.IntegerField()
+    groups_created = models.IntegerField()
+    run_date = models.DateTimeField()
+    total_user_logins = models.IntegerField()
+    total_visualization_launches = models.IntegerField()
+    total_workflow_launches = models.IntegerField()
+    users_created = models.IntegerField()
+    unique_user_logins = models.IntegerField()
+
+    class Meta:
+        verbose_name_plural = "site statistics"
+
+    def save(self, force_insert=False, force_update=False, using=None,
+             update_fields=None):
+        self.run_date = timezone.now()
```

Comment:
Set `auto_now=True` on run_date instead?
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.DateField.auto_now

Reply:
If I set `auto_now=True` on `run_date` I wouldn't have access to the value until after `save()` was complete. I need to know the `run_date` in the methods called within `save()`